cmd/consensus/main.go

```diff
@@ -602,9 +602,10 @@ func main() {
 				logger,
 				mainMetrics,
 			)
-
-			notifier.AddParticipantConsumer(telemetryConsumer)
 			notifier.AddFollowerConsumer(followerDistributor)
+			notifier.AddFinalizationConsumer(telemetryConsumer)
```

Comment:
What if we introduce a `TelemetryConsumer` interface as such:
```
type TelemetryConsumer interface {
	ParticipantConsumer
	CommunicatorConsumer
	FinalizationConsumer
	VoteCollectorConsumer
	TimeoutCollectorConsumer
}
```

and encapsulate logic of subscribing into distributor function `AddTelemetryConsumer`. Even though it's quite specialized I would say it looks cleaner and safer that way.